schedule/qam/common/qam-scc.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: qam-scc
+description: yast2 registration and SUSEConnect test
+schedule:
+- autoyast/prepare_profile
+- installation/isosize
+- installation/bootloader
+- autoyast/installation
+- autoyast/console
+- autoyast/autoyast_reboot
+- installation/grub_test
+- installation/first_boot
+- console/system_prepare
+- console/consoletest_setup
+- qa_automation/patch_and_reboot
+- console/yast2_registration
+- console/suseconnect
+...
```

Comment:
Is that on purpose? Looks unintentional.
Also quick questions, will it be parsed correctly without indentation for the list?

Reply:
Ah, have found that it's in the standard, learnt something! So ignore first question =)

Reply:
not sure if you mean `...`, but yes I like to use it as we yal does start with `---` and making it end with `...` make end obvious
Added indentation, many yamls don't have any indentation

Reply:
JFYI, so called "zero-indented" sequences will be parsed by all known YAML parsers (haven't found one that doesn't (except the old YAML.pm which I fixed ;-))